manifests/m/Mariocube/BannerEditingTool/0.8/Mariocube.BannerEditingTool.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.BannerEditingTool
+PackageVersion: 0.8
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Wii/PC%20Applications/Banner%20Editing%20Tools/tools%20v0.8.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: nullsoft

```

```suggestion
  NestedInstallerType: exe
```
`tools\AudacityPortable.exe` isn't `nullsoft` since it's not an installer.

It looks like a portable app to me when I tested in a VM so launching `tools\AudacityPortable.exe` without any arguments launches successfully.